package.json

```diff
@@ -31,8 +31,8 @@
     "generate:meta": "node --loader @polkadot/dev-ts node_modules/.bin/polkadot-types-from-chain --endpoint metadata.json --package polymesh-types --output ./src/polkadot --strict",
     "generate:consts": "cross-env node scripts/generateConsts.js",
     "generate:post-process": "cross-env node scripts/postProcessTypes.js",
-    "test": "jest --coverage",
-    "test:no-cov": "jest",
+    "test": "POLKADOTJS_DISABLE_ESM_CJS_WARNING=1 jest --coverage",
```

Comment:
This will prevent this polkadot warnings from cluttering the test output.

They should still show up if the packages have different paths though.